daprdocs/content/en/dotnet-sdk-docs/dotnet-workflow/dotnet-workflow-howto.md

```diff
@@ -105,21 +105,21 @@ curl -i -X POST http://localhost:3500/v1.0-alpha1/workflows/dapr/OrderProcessing
 If successful, you should see a response like the following: 
 
 ```json
-{"instance_id":"1234"}
+{"instance_id":"12345678"}
 ```
 
 Send an HTTP request to get the status of the workflow that was started:
 
 ```bash
-curl -i -X GET http://localhost:3500/v1.0-alpha1/workflows/dapr/OrderProcessingWorkflow/1234
+curl -i -X GET http://localhost:3500/v1.0-alpha1/workflows/dapr/12345678
 ```
 
 The workflow is designed to take several seconds to complete. If the workflow hasn't completed when you issue the HTTP request, you'll see the following JSON response (formatted for readability) with workflow status as `RUNNING`:
 
 ```json
 {
   "WFInfo": {
```

Comment:
We'll need to update this entire JSON payload here since it's changing with v1.11.